src/coreclr/vm/codeman.cpp

```diff
@@ -1838,6 +1838,18 @@ static void LoadAndInitializeJIT(LPCWSTR pwzJitName DEBUGARG(LPCWSTR pwzJitPath)
     }
 }
 
+#ifdef FEATURE_INTERPRETER
+void InitializeInterpreter(IN HINSTANCE phJit, OUT ICorInterpreter **pNewInterpreter)
+{
+    typedef ICorInterpreter* (__stdcall* pGetInterpreterFn)();
```

Comment:
if we mark this as __stdcall it should be synced with the actual implementation (that I believe didn't have any attributes).

Reply:
Another copy&paste from the historic JIT/EE interface setup. It would be best to clean it up. For internal implementation details like this one, we prefer to omit explicit calling conventions.